its/src/test/java/its/ConnectedModeTest.java

```diff
@@ -179,6 +179,27 @@ public void updateProject() throws Exception {
   }
 
   @Test
+  public void verifyExtendedDescription() throws Exception {
+    updateGlobal();
+
+    String ruleKey = "squid:S106";
+
+    assertThat(engine.getRuleDetails(ruleKey).getExtendedDescription()).isEmpty();
+
+    String extendedDescription = "my dummy extended description";
+
+    WsRequest request = new PostRequest("/api/rules/update")
+      .setParam("key", ruleKey)
+      .setParam("markdown_note", extendedDescription);
+    WsResponse response = adminWsClient.wsConnector().call(request);
+    assertThat(response.code()).isEqualTo(200);
```

Comment:
I'd like that, but `adminWsClient` (instance of `WsClient`) doesn't have a `post` method...